tests/test_oauth.py

```diff
@@ -339,17 +340,21 @@ def test_oauth_version_default(self):
         # OAuth 2.0 is the default version
         assert_is(self.provider._oauth_version, OAUTH2)
 
-    def test_start_flow(self):
-        # Generate the appropriate URL and state token
+    def test_start_flow_oauth_standard(self):
+        # Generate the appropriate URL and state token - addons that follow standard OAuth protocol
+
+        # Make sure that the mock oauth2 provider is a standard one.  The test would fail early here
+        # if `test_start_flow_oauth_no_redirect()` was ran before this test and failed in the middle
+        # without resetting the `ADDONS_OAUTH_NO_REDIRECT` list.
+        assert self.provider.short_name not in ADDONS_OAUTH_NO_REDIRECT
```

Comment:
All changes except this line in `test_start_flow_oauth_standard()` are non-functional style improvements.

Reply:
thanks for clarifying 👍